api/modules/checklist/model.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class Checklist(models.Model):
```

Comment:
What is the need for this Model?
If this serves the purpose of multiple checklists for same user, there should be an option to have name for each checklist and while fetching all items in a checklist, you would have to take input the name/ID of checklist. Also you would need to support an API to a list of all checklists.